com.outerwildsmods.owmods_gui.yml

```diff
@@ -0,0 +1,82 @@
+app-id: com.outerwildsmods.owmods_gui
+runtime: org.gnome.Platform
+runtime-version: '44'
+sdk: org.gnome.Sdk
+sdk-extensions:
+    - org.freedesktop.Sdk.Extension.mono6
+command: outer-wilds-mod-manager
+rename-icon: outer-wilds-mod-manager
+finish-args:
+    - --socket=fallback-x11
+    - --socket=wayland
+    - --share=ipc
+    - --share=network
+    - --filesystem=home
+    - --filesystem=/tmp
+    - --device=dri
+modules:
+  - 'shared-modules/libsoup/libsoup-2.4.json'
+  - name: webkitgtk
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=None
+      - -DPORT=GTK
+      - -DUSE_SOUP2=ON
+      - -DUSE_LIBSECRET=OFF
+      - -DUSE_WOFF2=OFF
+      - -DENABLE_BUBBLEWRAP_SANDBOX=OFF
+      - -DENABLE_GAMEPAD=OFF
+      - -DENABLE_INTROSPECTION=OFF
+      - -DENABLE_SPELLCHECK=OFF
+    sources:
+      - type: archive
+        url: https://webkitgtk.org/releases/webkitgtk-2.38.5.tar.xz
+        sha256: 40c20c43022274df5893f22b1054fa894c3eea057389bb08aee08c5b0bb0c1a7
+        x-checker-data:
+          type: html
+          url: https://webkitgtk.org/releases/
+          version-pattern: LATEST-STABLE-(\d[\.\d]+\d)
+          url-template: https://webkitgtk.org/releases/webkitgtk-$version.tar.xz
+      - type: shell
+        commands:
+          - sed -i 's@NAMES avif.h@NAMES avif/avif.h@' Source/cmake/FindAVIF.cmake
+          - sed -i '/PATH_SUFFIXES avif/d' Source/cmake/FindAVIF.cmake
+          - sed -i 's@${AVIF_INCLUDE_DIR}/avif.h@${AVIF_INCLUDE_DIR}/avif/avif.h@'
+            Source/cmake/FindAVIF.cmake
+  - name: openssl11
```

Comment:
do you really need this old version of openssl?

Reply:
Yes it's a requirement imposed by Tauri's HTTP client